web/views/error_view.ex

```diff
@@ -9,6 +9,14 @@ defmodule Org.ErrorView do
     "Internal server error"
   end
 
+  def render("401.json", _assigns) do
+    %{error: "Please sign in"}
```

Comment:
Oh I'm fine with the generic HTTP status messages. I just kept what was there before when trying to access a page without being signed in. Hmmm, I wonder if we need to return the status like that? Or if Phoenix knows to return that already?

But yeah, I'm fine with formatting it with `{data: ..., status: ..., message: ...}`